src/App.jsx

```diff
@@ -14,57 +16,74 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setValue] = useState('Jam');
```

Comment:
lets create a helper her
```suggestion
  const [selectedGood, setValue] = useState('Jam');
  const isSelected = good => selectedGood === good;
```